account/src/main/java/bisq/account/protocol_type/LoanProtocolType.java

```diff
@@ -20,15 +20,18 @@
 import bisq.common.util.ProtobufUtils;
 
 public enum LoanProtocolType implements ProtocolType {
+    UNSPECIFIED,
```

Comment:
We do not want to have the `UNSPECIFIED` entry in the java code.

Reply:
I changed this further to remove the hardcoded prefix name and use the class simple_name to get it